test/qa/lib/daemon.py

```diff
@@ -188,3 +193,19 @@ def get_status_data() -> dict:
     colon_separated_pairs = (element.split(':') for element in lines)
     formatted_pairs = {(key.lower(), value.strip()) for key, value in colon_separated_pairs}
     return dict(formatted_pairs)
+
+def get_env() ->str:
+    """
+    Detects and returns the active environment (DEV or PROD) based on the NordVPN version output.
+
+    :return: the active environment (DEV or PROD)
+    """
+    result = subprocess.run(
```

Comment:
the rest of the file is using `sh` library to run bash commands in regards to nordvpn, maybe it makes sense to either use `sh` for all or `subprocess` for all and not mix and match